Templates/BlobTrigger/readme.md

```diff
@@ -0,0 +1,11 @@
+# BlobTrigger - Node
+
+The `BlobTrigger` makes it incredibly easy to react to new Blobs inside of Azure Blob Storage. This sample demostrates a simple use case of processing data from a given Blob using Node.
```

Comment:
"demostrates" spelled wrong. Perhaps we can run spell check on this stuff eventually? :)